contentcuration/contentcuration/frontend/channelEdit/views/progress/ProgressModal.vue

```diff
@@ -104,13 +138,26 @@
     beforeDestroy() {
       clearInterval(this.timer);
     },
+    methods: {
+      ...mapActions(['showSnackbar']),
+      showSnackbarOnCompleteSync(showProgress) {
+        if (showProgress && this.nodeId) {
+          ContentNode.waitForCopying([this.nodeId]).then(() => {
```

Comment:
Ah! I mentioned `waitForCopying` as an example for how you could wait for the sync task. I wouldn't expect it to work for syncing because it checks the existence of `COPYING_FLAG` which isn't applicable to a sync.

Since you implemented a `watch`er to trigger the snackbar, that seems sufficient. So either use that or implement a `waitForTask` and check the existence of the `TASK_ID` flag. You shouldn't need both!